src/neuroglancer/segmentation_graph/source.ts

```diff
@@ -18,13 +18,22 @@ import {VisibleSegmentsState} from 'neuroglancer/segmentation_display_state/base
 import {WatchableValueInterface} from 'neuroglancer/trackable_value';
 import {Disposer, Owned, RefCounted} from 'neuroglancer/util/disposable';
 import {Uint64} from 'neuroglancer/util/uint64';
+import {RenderLayer } from 'neuroglancer/renderlayer';
+import {RenderLayerTransformOrError} from 'neuroglancer/render_coordinate_transform';
+import {MultiscaleVolumeChunkSource} from 'neuroglancer/sliceview/volume/frontend';
+
+export enum VISIBLE_SEGMENT_TYPE {
```

Comment:
Can you change to `VisibleSegmentType` and add a brief comment to describe the behavior of each one?

Reply:
Pinging again regarding this.